library/ctr_drbg.c

```diff
@@ -90,15 +90,13 @@ int mbedtls_ctr_drbg_seed_entropy_len(
     /*
      * Initialize with an empty key
      */
-    if( ( ret = mbedtls_aes_setkey_enc( &ctx->aes_ctx, key, MBEDTLS_CTR_DRBG_KEYBITS ) ) != 0 )
-    {
-        return( ret );
-    }
+    if( ( ret = mbedtls_aes_setkey_enc( &ctx->aes_ctx,
+                                        key, MBEDTLS_CTR_DRBG_KEYBITS ) ) != 0 )
+            return( ret );
```

Comment:
Style: Could you please reduce the indentation here by 1 level?

Reply:
done